pkg/controller/pcideviceclaim/pcideviceclaim_controller.go

```diff
@@ -100,118 +166,212 @@ func unbindPCIDeviceFromVfioPCIDriver(addr string) error {
 	return nil
 }
 
-func (h Handler) reconcilePCIDeviceClaims(nodename string) error {
-	// Get all PCI Device Claims
-	var pdcNames map[string]string = make(map[string]string)
-	pdcs, err := h.pdcClient.List(metav1.ListOptions{})
+func addHostDeviceToKubeVirtAllowList(pd *v1beta1.PCIDevice) error {
+	logrus.Infof("Adding %s to KubeVirt Allow-List", pd.Name)
+	clientConfig := kubecli.DefaultClientConfig(&pflag.FlagSet{})
+	virtClient, err := kubecli.GetKubevirtClientFromClientConfig(clientConfig)
```

Comment:
this should be configured on the mian.go and pass through to the controller's handler accordingly to avoid multiple initializations.

Reply:
I agree, although I am going to push it down from main.go to the Register function for PCIDeviceClaims controller. They have different handlers, the PCIDeviceClaims controller has three different clients now, including the KubeVirt one. The PCIDevices controller is very simple, it just maintains the PCIDevice objects in accordance with what it's node's list of PCI Devices contains.